cmd/neofs-cli/modules/control/object_list.go

```diff
@@ -0,0 +1,54 @@
+package control
+
+import (
+	rawclient "github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	"github.com/spf13/cobra"
+)
+
+var listObjectsCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List of all objects in the storage node",
+	Long:  "List of all objects in the storage node",
+	Args:  cobra.NoArgs,
+	Run:   listObjects,
+}
+
+func initControlObjectsListCmd() {
+	initControlFlags(listObjectsCmd)
+}
+
+func listObjects(cmd *cobra.Command, _ []string) {
+	ctx, cancel := commonflags.GetCommandContext(cmd)
+	defer cancel()
+
+	pk := key.Get(cmd)
+
+	req := &control.ListObjectsRequest{
+		Body: &control.ListObjectsRequest_Body{},
+	}
+	signRequest(cmd, pk, req)
+
+	cli := getClient(ctx, cmd)
+
+	var resp []*control.ListObjectsResponse
```

Comment:
my stream request was mostly about memory consumption and user experience: if we received some info, we can print it and drop it from memory. if we are about to download e.g. 100mb it will take seconds of no output for a user. can you try not to hold every streamed message in memory?

Reply:
I thought about it, but I couldn't find a good solution. Now I've done it using the function inside.